src/script/conversation/ConversationRepository.ts

```diff
@@ -616,27 +617,36 @@ export class ConversationRepository {
   ): Promise<ContentMessage[]> {
     const hasAdditionalMessages = events.length === Config.getConfig().MESSAGES_FETCH_LIMIT;
 
+    const isMemberMessage = (
+      message: Message | ContentMessage | MemberMessage | SystemMessage,
+    ): message is MemberMessage => message.super_type === SuperType.MEMBER;
+
     const mappedMessageEntities = await this.addEventsToConversation(events, conversationEntity);
     conversationEntity.hasAdditionalMessages(hasAdditionalMessages);
     if (!hasAdditionalMessages) {
-      const firstMessage = conversationEntity.getFirstMessage() as MemberMessage;
-      const checkCreationMessage = firstMessage?.isMember() && firstMessage.isCreation();
-      if (checkCreationMessage) {
-        const groupCreationMessageIn1to1 = conversationEntity.is1to1() && firstMessage.isGroupCreation();
-        const one2oneConnectionMessageInGroup = conversationEntity.isGroup() && firstMessage.isConnection();
```

Comment:
This needed a change because we couldnt be sure that the first message is always defined / a member message. isCreation() has been undefined for some reloads which caused the if statements to break -> unwanted side effects